pymc/distributions/distribution.py

```diff
@@ -601,6 +656,22 @@ def update(self, node: Node):
         return updates
 
 
+@_moment.register(CustomSymbolicDistRV)
+def dist_moment(rv, *args):
```

Comment:
First argument in now op when you register. It also means you don't need to recover it from the node inside

```suggestion
def dist_moment(op, rv, *args):
```

Reply:
Yeah, thank you!